recipe/meta.yaml

```
@@ -24,78 +24,76 @@ requirements:
     - python-build
     - meson-python ==0.13.1
     - meson ==1.2.1
-    - cython >=0.29.33,<3
+    - cython >=3.0.5
     - numpy {{ numpy }}
     - versioneer
-    - tomli # [py<311]
     - pip
   run:
     - python
     # Base Dependencies
     - {{ pin_compatible('numpy') }}
     - python-dateutil >=2.8.2
     - pytz >=2020.1
-    - python-tzdata >=2022.1
+    - python-tzdata >=2022.7
     # Recommended dependencies to achieve large speedups
     # see https://pandas.pydata.org/pandas-docs/version/2.1.1/getting_started/install.html#performance-dependencies-recommended
-    - numexpr >=2.8.0
-    - bottleneck >=1.3.4
+    - numexpr >=2.8.4
+    - bottleneck >=1.3.6
   run_constrained:
     # Optional dependencies
     # see https://pandas.pydata.org/pandas-docs/version/2.1.1/getting_started/install.html#optional-dependencies
     # and https://github.com/pandas-dev/pandas/blob/v2.1.1/pyproject.toml
     # performance
-    - numba >=0.55.2
+    - numba >=0.56.4
     # plot
-    - matplotlib-base >=3.6.1
+    - matplotlib-base >=3.6.3
     # output_formatting
     - jinja2 >=3.1.2
-    - tabulate >=0.8.10
+    - tabulate >=0.9.0
     # computation
-    - scipy >=1.8.1
-    - xarray >=2022.03.0
+    - scipy >=1.10.0
+    - xarray >=2022.10.0
     # fss
-    - fsspec >=2022.05.0
+    - fsspec >=2022.11.0
     # aws
-    - s3fs >=2022.05.0
+    - s3fs >=2022.11.0
     # gcp
-    - gcsfs >=2022.05.0
-    - pandas-gbq >=0.17.5
+    - gcsfs >=2022.11.0
+    - pandas-gbq >=0.19.0
     # excel
     - odfpy>=1.4.1
-    - openpyxl >=3.0.10
-    - pyxlsb >=1.0.9
+    - openpyxl >=3.1.0
+    - pyxlsb >=1.0.10
     - xlrd >=2.0.1
-    - xlsxwriter >=3.0.3
+    - xlsxwriter >=3.0.5
     # parquet, feather
-    - pyarrow >=7.0.0
+    - pyarrow >=10.0.1
     # hdf5
-    - blosc >=1.21.0
-    - pytables >=3.7.0
+    - blosc >=1.20.1
```

```suggestion
    - blosc >=1.21.3
```
https://github.com/pandas-dev/pandas/blob/v2.2.1/pyproject.toml#L94

---

Done!